src/utils/querybuilder.util.js

```diff
@@ -514,16 +515,49 @@ const quantityQueryBuilder = function ({ target, field }) {
                 qB[`${field}.value`] = { $gte: num };
                 break;
             case 'ne':
-                qB[`${field}.value`] = { $ne: num };
+                offset = calcRange(num);
```

Comment:
Getting error in 'ne' prefix query, here we are passing num as number but expecting it to be string in calcRange
`offset = calcRange(num.toString());`

Reply:
Also add test case for this

Reply:
Bailed and created EFS-1141 to address it.